miniwin/sdl3gpu/src/miniwin_d3drm.cpp

```diff
@@ -294,6 +306,19 @@ HRESULT Direct3DRM_SDL3GPUImpl::CreateViewport(
 		return DDERR_GENERIC;
 	}
 
+	SDL_GPUTextureCreateInfo depthTextureInfo = {};
+	depthTextureInfo.type = SDL_GPU_TEXTURETYPE_2D;
+	depthTextureInfo.format = SDL_GPU_TEXTUREFORMAT_D16_UNORM;
```

Comment:
Do you know what depth the original used? It seems it will use the highest offered of 8/16/24/32

Reply:
No, I just tried [`IDirect3DViewport::GetBackgroundDepth`](https://library.thedatadungeon.com/directx5/html/dire9e7g.htm) in the d3drm example, but I get a NULL surface.